internal/config/config.go

```diff
@@ -64,10 +65,11 @@ type API struct {
 }
 
 type EKS struct {
-	AccountID   string        `mapstructure:"account_id"`
-	Region      string        `mapstructure:"region"`
-	ClusterName string        `mapstructure:"cluster_name"`
-	APITimeout  time.Duration `mapstructure:"api_timeout"`
+	AccountID                         string        `mapstructure:"account_id"`
+	Region                            string        `mapstructure:"region"`
+	ClusterName                       string        `mapstructure:"cluster_name"`
+	APITimeout                        time.Duration `mapstructure:"api_timeout"`
+	APINodeLifecycleDiscoveryDisabled *bool         `mapstructure:"api_node_lifecycle_discovery_disabled"`
```

Comment:
**nitpick**, usually we avoid setting `negative` style bool configs and opt in for APINodeLifecycleDiscoveryEnabled with sane defaults instead